compiler+runtime/src/cpp/jank/read/parse.cpp

```diff
@@ -1215,7 +1206,9 @@ namespace jank::read::parse
           auto const resolved_ns(__rt_ctx->resolve_ns(make_box<obj::symbol>(ns_portion)));
           if(resolved_ns.is_none())
           {
-            return err(error{ token.pos, fmt::format("unknown namespace: {}", ns_portion) });
+            return error::parse_unresolved_namespace(
+              fmt::format("Unresolved namespace '{}'", ns_portion),
+              { start_token.start, latest_token.end });
```

Comment:
FYI this case might not be an error due to https://github.com/jank-lang/jank/issues/195, but I'm working on that.

I have this as

```
            ns = ns_portion;
```

in my WIP branch, but I don't have an example handy:

https://github.com/jank-lang/jank/pull/239/files#diff-bced15263f03bb66a379b6f9346fa47a66e15b168e39a710e7153d1c115090e2R1222

Reply:
Gotcha. Leaving as-is for now. Can be fixed in your PR.